lib/helpers.bash

```diff
@@ -326,6 +337,7 @@ _disable-plugin ()
     _group 'lib'
 
     _disable-thing "plugins" "plugin" $1
+    _on-disable-callback $1
```

Comment:
This is great - love this idea! I think it makes sense to provide a note about this in the `DEVELOPMENT.md` file.

Reply:
done